content/archive/_index.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: "Archive"
-description: "The CHipCie Problemset Archive"
+linkTitle: "Archive"
+title: "The CHipCie Problem Set Archive"
```

Comment:
Good change to split up "Problemset" into "Problem Set", thanks :smile: Can you cherry-pick mpsijm@7ad051a623b0baac2e485f213dd17afff3c4e08c? I've applied this change in other places as well :slightly_smiling_face:

Reply:
Done :)